settings/src/components/webauthn/list-keys.js

```diff
@@ -95,12 +95,15 @@ export default function ListKeys() {
 function ConfirmRemoveKey( { keyToRemove, onConfirm, onClose, deleting, error } ) {
 	return (
 		<Modal
-			title={ `Remove ${ keyToRemove.name }?` }
+			title="Delete security key"
 			className="wporg-2fa__confirm-delete-key"
 			onRequestClose={ onClose }
 		>
 			<p className="wporg-2fa__screen-intro">
-				Are you sure you want to remove the <code>{ keyToRemove.name }</code> security key?
+				Are you sure you want to delete the following key?
+				<ul className="wporg-2fa__screen-list">
```

Comment:
Why a list? I get that you're after a bulleted style, but semantically it doesn't make sense to me if there's only ever going to be one item. Maybe a key icon or some other treatment could work?

Reply:
Yeah, understood. It was the easiest approach. 

There's a small chance that this could be used for deleting multiple keys in the future but that definitely isn't set in stone. I can change it to something else if it doesn't make sense.

Reply:
The key icon is pretty nice.

<img width="373" alt="Screenshot 2023-09-13 at 2 41 06 PM" src="https://github.com/WordPress/wporg-two-factor/assets/1657336/be8902e3-ad66-4f83-abab-f452ccd3cae3">